src/poly/polymorphisem/PolyHelperClass.java

```diff
@@ -0,0 +1,32 @@
+package poly.polymorphisem;
+
+import java.util.Arrays;
+
+public class PolyHelperClass {
+    public static void basketballPlayerAgenda(BasketballPlayer[] players) {
+        for (BasketballPlayer player : players) {
+            if (player instanceof Fish) {
+                ((Fish) player).goToWork();
```

Comment:
```suggestion
            if (player instanceof Fish fishPlayer) {
                fishPlayer.goToWork();
```
instead of casting you can create a pointer to player that is type Fish (presentation page 24)